campos_potenciais2.py

```
@@ -0,0 +1,278 @@
+    res = M @ p
+    return res  # 3-vector
+
+# ---------------------------------------------------- Conexão com CoppeliaSim ---------------------------------------------------------------
+client = RemoteAPIClient()
+sim = client.require("sim")
+sim.setStepping(True)
+
+
+robotHandle = sim.getObject("/" + ROBOT_NAME) if sim.getObject("/" + ROBOT_NAME) != -1 else sim.getObject(ROBOT_NAME)
+if robotHandle == -1:
+    raise RuntimeError(f"Handle do robô não encontrado: verifique ROBOT_NAME='{ROBOT_NAME}' na sua cena.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)